src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/ProjectPropertiesItemHandler.cs

```diff
@@ -21,6 +21,16 @@ public ProjectPropertiesItemHandler(IWorkspaceProjectContext context)
             _context = context;
         }
 
+        public string EvaluationRule
+        {
+            get { return ConfigurationGeneral.SchemaName; }
+        }
+
+        public void Initialize(IWorkspaceProjectContext context)
+        {
+            _context = context;
```

Comment:
It's expected that we can overwrite the once given in the constructor now?

Reply:
Ah, problem is supporting both language services hosts; one grabs them via MEF, the other manually creates them. I'll add an assert.